lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua

```diff
@@ -0,0 +1,126 @@
+CFCE2Lib.RegisterExtension( "cfc_e2_lib", true )
+
+-- Made by StrawWagen
+
+-- Setting up local functions 
+
+local function isValidPly( ply )
+    if not ply or not ply:IsValid() or not ply:IsPlayer() then
+        return false
+    end
+    return true
+end
+
+local function isValidNpc( npc )
+    if not npc or not npc:IsValid() or not npc:IsNPC() then
+        return false
+    end
+    return true
+end
+
+
+local function hasAccess(ply)
+    return ply:IsAdmin()
+end
+
+
+-- Admin only npc functions
+
+e2function void entity:npcSetMaxHealth( number maxhealth )
+    if not ValidNpc( this ) then return end
+    if not ValidPly( self.player ) then return end
```

Comment:
You'll need to go through and update these calls now that you changed the name of the validity check. (Or you could just replace them with `IsValid` and never worry about that again :) )